server/controllers/stock/inventory.js

```diff
@@ -60,6 +69,40 @@ exports.getInventoryDonations = getInventoryDonations;
 exports.getInventoryDonationsById = getInventoryDonationsById;
 
 /**
+ * POST /inventory/metadata
+ * Create a new inventory data entry
+ */
+function createInventoryItems(req, res, next) {
+  'use strict';
+
+  core.createItemsMetadata(req.body, req.session)
+  .then((identifier) => {
```

Comment:
So that you know, when there is a single parameter, the parentheses are optional.  The following are equivalent:
```js
.then((identifier) => {
  res.status(201).json({ uuid : identifier });
})
```
... and ...
```js
.then(identifier => {
  res.status(201).json({ uuid: identifier});
});
```

It is your decision which one to use - I have no preference.  But just so you know it is available :)